speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
It's cool, and typically when I am reporting back on things I have tested I don't use my local branch. 

> I'm not trying to argue that you should populate your slots to get it to work

Me neither :) and I do acknowledge that if it was in my slot chances are it would have worked fine..... 

.. though to also satisfy my own curiosity I have just removed the articles so the samples were back to 

```
WatchMovie play film {Movie}
WatchMovie play movie {Movie}
WatchMovie watch film {Movie}
WatchMovie watch movie {Movie}
```

and added ghostbusters to my MOVIES slot. 

The results are more confusing, as it didn't work, it is still looking for "the movie ghostbusters" 

```
Trying to match: the movie ghostbusters
Simple match failed, trying fuzzy match...
```

I thought there may have been a cache issue with the Skill but realised if there was then it would have still worked as it was working on the last "build". 

So that is very interesting. It must be due to the learning or the UK/US differences. I would rather not reset my Echo and erase the voice history to find out though :)